components/edit-collective/sections/ThankYouEmail.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useMutation } from '@apollo/client';
+import { FormattedMessage } from 'react-intl';
+
+import { CollectiveType } from '../../../lib/constants/collectives';
+import { API_V2_CONTEXT, gqlV2 } from '../../../lib/graphql/helpers';
+
+import Container from '../../Container';
+import { Box, Flex } from '../../Grid';
+import MessageBox from '../../MessageBox';
+import RichTextEditor from '../../RichTextEditor';
+import StyledButton from '../../StyledButton';
+import StyledCheckbox from '../../StyledCheckbox';
+import StyledHr from '../../StyledHr';
+import { P } from '../../Text';
+import { TOAST_TYPE, useToasts } from '../../ToastProvider';
+
+const AdditionalSettingsCheckBox = ({ onChange, checked }) => {
+  return (
+    <StyledCheckbox
+      onChange={({ checked, name }) => onChange({ target: { name, value: checked } })}
+      checked={checked}
+      name="additionalSettings"
+      label={
+        <FormattedMessage defaultMessage="Inherit the above collective level custom message to all tiers, event and projects if no custom message is set at the respective levels." />
+      }
+    />
+  );
+};
+
+AdditionalSettingsCheckBox.propTypes = {
+  onChange: PropTypes.func,
+  checked: PropTypes.bool,
+};
```

Comment:
Comment if not used.

Reply:
Ops, something I missed removing after the updates to the design. I've removed this. 👍🏽